versioned_docs/version-1.1.x/how-to/connect-to-surrealdb-via-ngrok-tunnel.mdx

```diff
@@ -0,0 +1,81 @@
+## Connect to SurrealDB via Ngrok tunnel
+
+This comprehensive guide will walk you through connecting a local SurrealDB instance to the internet using Ngrok, making it accessible remotely. 
+Ngrok is **a cross-platform application that allows developers to expose their local web servers to the internet**. It hosts a local web server on its own sub-domain and makes your local development box available on the internet through Tunneling.
+
+This setup is beneficial for remote development and quick testing of applications built locally on SurrealDB. 
+
+### Prerequisites
+
+This guide assumes the following:
+
+- You have [SurrealDB installed](https://surrealdb.com/install) and a [fresh instance of SurrealDB running.](https://docs.surrealdb.com/docs/introduction/start)
```

Comment:
Within docs, we use the relative paths  so `https://docs.surrealdb.com/docs/introduction/start` will be `/docs/introduction/start`